e2e/test/MsTestLogger.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using System;
+using System.Collections.Generic;
+using System.Runtime.CompilerServices;
+using System.Text;
+using System.Threading;
+using Microsoft.ApplicationInsights.DataContracts;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Newtonsoft.Json;
+
+namespace Microsoft.Azure.Devices.E2ETests
+{
+    public class MsTestLogger : TestLogging
```

Comment:
Calling this MsTestLogger because we can have a mix of MsTest and XUnit. This should help add MsTest sepecific implementation to get the test related data.